src/main/java/io/stargate/sgv2/jsonapi/config/DocumentLimitsConfig.java

```diff
@@ -42,6 +45,10 @@ public interface DocumentLimitsConfig {
   @WithDefault("64")
   int maxObjectProperties();
 
+  @Positive
```

Comment:
Could you add Javadoc to explain limit, similar to how most other entries have it for `@return`?